src/mesh/impls/bout/boutmesh.cxx

```diff
@@ -2236,9 +2236,9 @@ void BoutMesh::addBoundaryRegions() {
     xe = -2;
   }
   
-  addRegion3D("RGN_UPPER_INNER_Y", Region<Ind3D>(xs, xe, 0, ystart-1, 0, LocalNz-1,
+  addRegion3D("RGN_UPPER_INNER_Y", Region<Ind3D>(xs, xe, yend, LocalNy-1, 0, LocalNz-1,
```

Comment:
Shouldn't this be `yend+1, LocalNy-1`, so that the region only includes guard cells, and not the last column of grid cells?

Reply:
Ah yes, because `yend` is the last interior point, not one-past-the-end. Cheers!